ada_2_fhir-r4/cio/2.0.0/beschikbaarstellen_contraindicatie/fhir_instance/mci-MCI-zwangerschap-1.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Misschien mis ik context, maar ik vind het een 'vreemd' scenario. De twee zwangerschappen die helemaal niet logisch zijn. Waar zwangerschap-2 later is geregistreed door dezelfde arts, en deze niet al is afgesloten ..

Reply:
Goede observatie! Bij CiO en MP gebruiken we FHIR eigenlijk op een 'onFHIRse' manier: bij élke wijziging kopieer je de hele resource, doe je de wijzigingen in de nieuwe resource en verwijs je naar de vorige instantie (normaal gesproken zou je in FHIR een nieuwe versie van dezelfde instantie aanmaken, en kun je altijd nog de history opvragen van de resource). Dit wordt zo gedaan omdat het in HL7v3 nu eenmaal zo werkt, en het ook voor de bronsystemen (blijkbaar) makkelijker is om telkens nieuwe instanties aan te maken.